api/reference/security_groups.adoc

```diff
@@ -0,0 +1,136 @@
+
+[[security-groups]]
+== Security Groups
+
+Management of Security Groups is available via the following endpoint:
```

Comment:
NIT, but the documentation fluctuates between 'via the following endpoint' and 'via the following collection'. Possibly keep it consistent between one or the other?